### PR TITLE
Fixing #1603 Add openssh-client to Docker image for git+ssh support

### PR DESCRIPTION
**Pull request by Lowess**
This PR addresses the issue https://github.com/ansible/molecule/issues/1603

#### PR Type

- Bugfix Pull Request

### CODE REVIEW

**Comment by lwm** [comment]
I know there has been debate on what and what not should be included in the Dockerfile but this looks like a no-brainer give the issue report which is clearly something that Molecule should be supporting. Thanks for bringing this change @Lowess.

**Comment by lwm** [comment]
It would perhaps be nice to include something with a `git+ssh` URL in the functional or scenario tests for this? I am not quite sure where it should go but it should be simple enough to find somewhere. Since the package is in transition from different maintenance, it would be nice to have tests for everything :)

**Comment by ragingpastry** [comment]
I think a separate functional test might be good. A large number of the current functional tests already bring in roles via ansible-galaxy so tacking on something with git+ssh might be a good solution. This would bring in external dependencies but I'm not sure how you would properly test this otherwise.

Perhaps the Docker build itself should be tested separately from Molecule proper?

**Comment by vutkin** [comment]
Hi @themr0c , @Lowess , @lwm we are supporting next docker images with systemd support for testing our roles: https://cloud.docker.com/u/leandelivery/repository/docker/leandelivery/docker-systemd

Could we use them as default molecule base images for testing? They already has all needed packages and improvements.
Will them work for us?

**Comment by themr0c** [comment]
@vutkin are these images publicly available ?

**Comment by tgadiev** [comment]
> @vutkin are these images publicly available ?
Yes, of course.
https://hub.docker.com/r/leandelivery/docker-systemd

**Comment by vutkin** [comment]
@themr0c sure, there are images for centos 7, ubuntu 16.04,18.04 with systemd support.

File: build/Dockerfile [hunk 1]
```diff
@@ -25,6 +25,7 @@ ENV PACKAGES="\
     ruby-rdoc \
     ruby-irb \
     git \
+    openssh-client
```
Comment (webknjaz) [inline build/Dockerfile]:
```suggestion
    openssh-client \
```

Comment (themr0c) [reply build/Dockerfile]: Hi @Lowess can you please apply the change suggested by @webknjaz ?

**Comment by Lowess** [comment]
Hi guys, sorry for my delayed response. I addressed the comment above but I wont have time right now to implement the test cases mentioned above by @vutkin. Can the testing improvement be addressed in an other PR ?

**Comment by lwm** [comment]
Agreed @Lowess, we'll torture you no longer :) 

@vutkin, can you please raise an issue for https://github.com/ansible/molecule/pull/1604#issuecomment-463749380, thanks!

**Comment by vutkin** [comment]
https://github.com/ansible/molecule/issues/1768

**Pull request merged**